jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -228,6 +228,50 @@
                   (send bbox :header :frame_id)
                   (send bbox :pose)))
       bx))
+  (:cube->cube-parallel-to-coords
+    (cube coords &rest args)
+    (let (copycoords target-ids cube-min-max-list)
+      (send cube :worldcoords)
+      ;; without this, body isn't updated
```

Comment:
Not weird.
`:worldcoords` update body.
In `(let (cube) (setq cube (make-cube 100 200 300)) (send cube :worldcoords) (send cube :locate #f(100 200 300)) (setq local-vertices (send cube :vertices)))`, you move `cube` after calling `:worldcoords`. So body of `cube` isn't changed.